presto-docs/src/main/sphinx/release/release-0.290.rst

```diff
@@ -0,0 +1,120 @@
+* Improve JoinPrefilter optimizer for wide join keys and multiple join keys. :pr:`23858`
+* Improve writer scaling in skewed conditions by setting ``optimized_scale_writer_producer_buffer`` to ``on`` by default. :pr:`23774`
+* Add UUID type support to the Parquet reader and writer. :pr:`23627`
+* Add a configurable-sized cache for Iceberg table puffin files to improve query planning time controlled by the  ``iceberg.max-statistics-file-cache-size`` configuration property. :pr:`23177`
+* Add a flag to the Presto CLI which allows skipping SSL certificate verification. :pr:`23780`
+* Add a session property ``native_max_extended_partial_aggregation_memory`` which specifies Presto native max partial aggregation memory when data reduction is optimal. :pr:`23527`
+* Add a session property ``native_max_partial_aggregation_memory`` which specifies Presto native max partial aggregation memory when data reduction is not optimal. :pr:`23527`
+* Add a session property ``native_max_spill_bytes`` which specifies Presto native max allowed spill bytes. :pr:`23527`
+* Add function :func:`is_private_ip` that returns true when the input IP address is private or a reserved IP address. :pr:`23520`
+* Add function :func:`ip_prefix_subnets` that splits the input prefix into subnets the size of the new prefix length. :pr:`23656`
+* Add new configuration property ``eager-plan-validation-enabled`` for eager building of validation of a logical plan before queuing. :pr:`23649`
+* Add session property ``inline_projections_on_values`` and configuration property ``optimizer.inline-projections-on-values`` to evaluate project node on values node. :pr:`23245`
+* Add support in QueuedStatement protocol to accept pre-minted query id and slug. :pr:`23407`
+* Add support to proxy AuthorizedIdentity using JWT. :pr:`23546`
+* Add support for casting ``char`` datatype to various numeric datatypes. :pr:`23792`
+* Replace configuration property ``async-cache-full-persistence-interval`` with ``async-cache-persistence-interval``. :pr:`23626`
+* Remove ``array_dupes`` and ``array_has_dupes`` alias names from functions :func:`array_duplicates` and :func:`array_has_duplicates`. :pr:`23762`
+
+Presto C++ Changes
+______________________________________
+* Fix ``task.writer-count`` and ``task.partitioned-writer-count`` configuration properties in Presto C++ for consistency with Presto. :pr:`23902`
+* Fix a bug where users weren't able to set the ``native_expression.max_array_size_in_reduce`` session property. :pr:`23856`
+* Fix plan validation failures for some join queries running with spill enabled when using Presto C++. :pr:`23595`
+* Improve arbitrator configs to use the new string-based format. :pr:`23496`
+* Add ``$path`` and ``$bucket`` to split info, and fixed the split counts in the coordinator UI. :pr:`23755`
+* Add configurable chunk size in ChunkedSliceOutput, reducing total memory usage by nearly 3x and per-object memory by 4x with a 1kB chunk size (default 8kB). :pr:`23707`
```

Comment:
I tried to understand how to toggle this, as it's written it's hard to understand how this relates to an operator of Presto. I recommend removing it or asking clarification from the author.

Reply:
Okay i asked the author didn't get a response yet. Won't want to block release notes for a PR. Also I think keeping in release notes is better if the people actually want to dig into code and see which operator it benefits but I am removing this entry

Reply:
Ideally this should be caught by whoever reviewed the PR.  Let's remove it if it can't be easily described as a user-facing feature.  CC: @chenyangfb @sdruzkin